code/modules/power/fusion/fuel_assembly/fuel_compressor.dm

```diff
@@ -54,7 +54,7 @@
 			to_chat(user, "<span class='warning'>It would be pointless to make a fuel rod out of [mat.use_name].</span>")
 			return
 		if(M.get_amount() < 15)
```

Comment:
Would probably be better to make this a constant or define somewhere while you're tweaking it.